.github/workflows/pr.yml

```diff
@@ -0,0 +1,53 @@
+name: PR
+on:
+  pull_request_target:
+    types:
+      - opened
+      - edited
+      - synchronize
+jobs:
+  main:
+    name: Validate PR title
+    runs-on: ubuntu-latest
+    steps:
+      - uses: amannn/action-semantic-pull-request@v5
+        id: lint_pr_title
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          # https://github.com/utilitywarehouse/uwos-go/blob/main/release-please-config.json#L53
```

Comment:
comment points at the wrong repo

Reply:
[I removed the comment](https://github.com/utilitywarehouse/shared-kustomize-bases/pull/41/commits/0262f3fa8ab3c0367a8a4ca97d1bc719ec894307#diff-15c806aa509538190832852f439e9921a23bec2da81f95ed0e4bf13c14e5b160L18) since I don't know what should it signify

Reply:
> [I removed the comment](https://github.com/utilitywarehouse/shared-kustomize-bases/pull/41/commits/0262f3fa8ab3c0367a8a4ca97d1bc719ec894307#diff-15c806aa509538190832852f439e9921a23bec2da81f95ed0e4bf13c14e5b160L18) since I don't know what should it signify

I think it's noting that this list is based off the entries in the `changelog-sections` in the release please config. If we wanted to we could probably just add a step to generate this list from the config file, but I imagine the list is not going to change much so :shrug: probably not worth the effort